frontend/src/screens/channels/first/FirstChannel.tsx

```diff
@@ -110,7 +110,7 @@ export function FirstChannel() {
             {new Intl.NumberFormat().format(channelSize)} sats of incoming
```

Comment:
Why is in the screenshot provided of this PR 100 sats instead of 100,000 sats in this phrase.

Reply:
Ah, when I was developing I mocked some values as the Megalith Mutinynet LSP service was down.